src/display/editor/editor.js

```diff
@@ -151,6 +151,7 @@ class AnnotationEditor {
       [
         "editor_alt_text_button_label",
         "editor_alt_text_decorative_tooltip",
+        "editor_alt_text_edit_button_label",
```

Comment:
Nit: Please move this up *one* line, such that we maintain the same ordering of as in the other files.